src/main/java/pro/trevor/tankgame/rule/definition/actions/DieRollResult.java

```diff
@@ -0,0 +1,24 @@
+package pro.trevor.tankgame.rule.definition.actions;
+
+import java.util.List;
+
+import org.json.JSONObject;
+
+import pro.trevor.tankgame.util.JsonType;
+
+@JsonType(name = "DieRollResult")
```

Comment:
How does this get encoded as JSON ever?

Reply:
No I think I originally thought it would though